docs/src/constants/redirect-routes.js

```diff
@@ -81,10 +81,13 @@ export default {
   '/components/native/text': ROUTES.TEXT,
   '/components/native/text-input': ROUTES.TEXT_INPUT,
   '/components/native/text-view': ROUTES.TEXT_VIEW,
-  '/components/native/tappable-link-label': ROUTES.TAPPABLE_LINK_LABEL,
+  '/components/native/tappable-link-label': ROUTES.LINK,
   '/components/native/touchable-native-feedback':
     ROUTES.TOUCHABLE_NATIVE_FEEDBACK,
   '/components/native/touchable-overlay': ROUTES.TOUCHABLE_OVERLAY,
+  '/components/star-rating-interactive': ROUTES.STAR_RATING,
+  '/components/tappable-link-label': ROUTES.LINK,
```

Comment:
This is a lie

Reply:
Given they are at different paths now they need redirect routes to put people in the correct place